aas_core_meta/v3rc1.py

```diff
@@ -242,7 +248,10 @@ def __init__(
 
 
 @abstract
-@reference_in_the_book(section=(4, 7, 2, 1))
+@reference_in_the_book(
+    section=(4, 7, 2, 1),
+    fragment="4.7.2.7%20Semantic%20References%20Attributes%20(" "HasSemantics))",
```

Comment:
```suggestion
    fragment="4.7.2.7 Semantic References Attributes (HasSemantics))",
```

Are you sure it's double parenthesis at the end?